package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "js-sdk-toolchain",
   "version": "7.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
which npm version are you using ?
there are a lot of changes for only updating a package version ? 
we need to verifiy that this update doesn't break anything, maybe its better to use npm 8 for now